client/modules/IDE/reducers/project.js

```diff
@@ -1,7 +1,10 @@
 import * as ActionTypes from '../../../constants';
+const generate = require('project-name-generator');
+
+const generatedName = generate({ words: 2 }).dashed;
 
 const initialState = {
-  name: 'Hello p5.js'
+  name: `${generatedName}`
```

Comment:
You should move the function call into the initial state, i.e. change this to 
```javascript
const initialState = {
    name: `${generate({ words: 2}).dashed}`
};
```
As it is now, the following happens, which I think is a bug:
1. User navigates to the editor and starts working on a project
2. User saves the project.
3. User clicks "New"
4. The new project has the same name as the previous project.